package.json

```diff
@@ -24,7 +24,7 @@
     "dev": "ts-node src/index.ts",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --verbose",
     "lint": "eslint --ext ts .",
-    "start": "node dist/index.js",
+    "start": "node dist/src/index.js",
```

Comment:
why don't we exclude the src folder from the path during the build?

Reply:
Yep, I'm in favour of `rootDir`

Reply:
Decided to keep this in order to have both the app and encryptor script available after build in the docker image